core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.html

```diff
@@ -503,10 +503,10 @@ <h3>Feedback/Suggestion Email Preferences</h3>
             </div>
           </div>
         </div>
-        <span class="form-text secondary-info-text" ng-if="!$ctrl.UserEmailPreferencesService.areFeedbackNotificationsMuted()">
+        <span class="form-text secondary-info-text" ng-if="$ctrl.UserEmailPreferencesService.areFeedbackNotificationsMuted()">
           You are currently receiving notifications of new feedback for this exploration.
         </span>
-        <span class="form-text secondary-info-text" ng-if="$ctrl.UserEmailPreferencesService.areFeedbackNotificationsMuted()">
+        <span class="form-text secondary-info-text" ng-if="!$ctrl.UserEmailPreferencesService.areFeedbackNotificationsMuted()">
```

Comment:
Hey @Sneakyhydra, although this solution achieves the goal of getting the description text to match the switch position, its not quite right.

The description itself is displayed just fine -- it's the switch that is displayed incorrectly. You should instead try to modify the switch's behaviour to fix this issue.

Take a look at the switch corresponding to this text (same with the suggestion emails) and see if you can figure out what the issue is. I think  it might be a case of the `checked` attribute being on the wrong checkbox causing the switch to be disabled when it should be enabled.

Reply:
@nithinrdy I have pushed the requested changes. Can you please review it?